crates/bevy_pbr/src/light.rs

```diff
@@ -425,6 +424,17 @@ pub fn update_directional_light_cascades(
                     transform.compute_matrix(),
                 ))
             }
+            (entity, transform, Projection::Orthographic(projection), camera)
+                if camera.is_active =>
+            {
+                Some((
+                    entity,
+                    ((projection.right - projection.left) / (projection.top - projection.bottom))
```

Comment:
```suggestion
                    ((projection.area.max.x - projection.area.min.x)
                        / (projection.area.max.y - projection.area.min.y))
```
maybe?

Reply:
yeah, but area is a `Rect` so `.width() / .height()` works.